src/analyses/call_graph_helpers.h

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************\
+
+Module: Function Call Graph Helpers
+
+Author: Chris Smowton, chris.smowton@diffblue.com
+
+\*******************************************************************/
+
+/// \file
+/// Function Call Graph Helpers
+
+#ifndef CPROVER_ANALYSES_CALL_GRAPH_HELPERS_H
+#define CPROVER_ANALYSES_CALL_GRAPH_HELPERS_H
+
+#include "call_graph.h"
+
+// These are convenience functions for working with the directed graph
+// representation of a call graph, obtained via
+// `call_grapht::get_directed_graph`. Usually function names must be mapped
+// to and from node indices, as in `graph.get_node_index("f")`, or
+// `graph[node_index].function`; these helpers include the translation for
+// convenience.
+
+/// Get functions directly callable from a given function
+/// \param graph: call graph
+/// \param function: function to query
+/// \return vector of called functions
+std::vector<irep_idt> get_successors(
```

Comment:
I'd suggest to use `std::set` or `std::unordered_set` -- otherwise it's not clear whether these will be unique or not. (They are not, but the ordering is completely arbitrary, which makes the result rather questionable.)